### DISCUSSION TITLE
How to create more than one database ?

### DISCUSSION THREAD

**Discussion by DadvDadv** [assuncaocharles/ngx-indexed-db#294] [Q&A]
Hi,
I d'nt figure how to create more than one database (not table), i would like to create a database by domains :

db1-users
   logs
   files
   
db2-catalog
   logs
   files
   items

and so on

Thx

**Comment by aparzi** [comment]
Hi @DadvDadv,
do you need to create two or more object stores or multiple db?

**Reply by DadvDadv** [reply]
multiple db

**Reply by DadvDadv** [reply]
i need to create a db (and all is object stores) for each Domain (user, catalog, products, etc.

Idealy i would like to create (or update) the db in a service, and not in the root module.
Then i could use some Authentification variable for create dynamic database (for example)

**Reply by aparzi** [reply]
Thanks @DadvDadv, understood. At the moment this thing is not possible but it is scheduled as a possible evolution.
I wanted to implement this feature as soon as possible, if you like you can open an issue from this thread, to use it as a reminder

**Reply by DadvDadv** [reply]
👍
